zebrad/tests/acceptance.rs

```diff
@@ -31,53 +32,76 @@ fn default_test_config() -> Result<ZebradConfig> {
     Ok(config)
 }
 
-#[derive(PartialEq)]
-enum ConfigMode {
-    NoConfig,
-    Ephemeral,
-    Persistent,
+fn persistent_test_config() -> Result<ZebradConfig> {
+    let mut config = default_test_config()?;
+    config.state.ephemeral = false;
```

Comment:
Let's also set the cache path to a subdirectory of the testdir, so that:
* we can run multiple tests in parallel
* these tests don't interfere with the generate tests, or other persistent tests

Here are the relevant lines from the old `ConfigMode::Persistent`:
```suggestion
    let cache_dir = dir.path().join("state");
    fs::create_dir(&cache_dir)?;
    config.state.cache_dir = cache_dir;
    config.state.ephemeral = false;
```

Reply:
Oh, I see, these changes ended up in `with_config`.

I'll resolve this now.